--- libvo/vo_x11.c.orig	Thu Dec 26 18:18:31 2002
+++ libvo/vo_x11.c	Thu Dec 26 19:58:49 2002
@@ -294,12 +294,12 @@
     bg=WhitePixel( mDisplay,mScreen );
     fg=BlackPixel( mDisplay,mScreen );
 
-    theCmap=vo_x11_create_colormap(&vinfo);
+    //theCmap=vo_x11_create_colormap(&vinfo);
 
     xswa.background_pixel=0;
     xswa.border_pixel=0;
-    xswa.colormap=theCmap;
-    xswamask=CWBackPixel | CWBorderPixel | CWColormap;
+    //xswa.colormap=theCmap;
+    xswamask=CWBackPixel | CWBorderPixel;// | CWColormap;
 
 #ifdef HAVE_XF86VM
     if ( vm )
@@ -321,6 +321,9 @@
     }
     else
      {
+    theCmap=vo_x11_create_colormap(&vinfo);
+    xswa.colormap=theCmap;
+    xswamask=CWBackPixel | CWBorderPixel | CWColormap;
       if ( vo_window == None )
        {
         vo_window=XCreateWindow( mDisplay,mRootWin,
